csi.proto

```diff
@@ -974,6 +983,21 @@ message ControllerGetVolumeResponse {
   // This field is REQUIRED.
   VolumeStatus status = 2;
 }
+message ModifyVolumeRequest {
```

Comment:
Like all other requests, please include a field for secrets, and some SPs will inevitably require it.

Reply:
Added ``map<string, string> secrets = 2 [(csi_secret) = true];``